lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -23,7 +23,8 @@ use JSON;
 use Fcntl;
 use DateTime;
 use db_helpers;
-use OpenQA::Utils qw(log_debug log_info log_warning parse_assets_from_settings locate_asset send_job_to_worker);
+use OpenQA::Utils
+  qw(log_debug log_info log_warning parse_assets_from_settings locate_asset send_job_to_worker read_test_modules);
```

Comment:
I guess a line break within the 'qw()' part would look nicer

Reply:
Actually we could also just use tags too, as right now we are exporting 40 of the 42 routines in the OpenQA::Utils, having `:test_handling` or `:logging` or `:scheduling` etc to group a bit might come handy... or simply start extracting from Utils... 

I'm going with @kraih's proposal

```
use OpenQA::Utils (
    qw(log_debug log_info log_warning),
    qw(parse_assets_from_settings locate_asset),
    qw(send_job_to_worker read_test_modules)
);
```

Reply:
Export tags seem like a good idea.